components/movementsensor/wheeledodometry/wheeledodometry_test.go

```diff
@@ -264,6 +265,7 @@ func TestSpin(t *testing.T) {
 		baseWidth:          0.2,
 		base:               base,
 		timeIntervalMSecs:  500,
+		originCoord:        geo.NewPoint(0, 0),
 	}
 	od.motors = append(od.motors, motorPair{left, right})
 	od.trackPosition(context.Background())
```

Comment:
Well, shoot. This line (and similar ones in other tests) used to spawn extra goroutines, and now does not, causing the tests to hang. Maybe each test needs its own StoppableWorker at this point? This mirrors something @abe-winter discussed with me, in which many tests spawn goroutines that they don't clean up properly, resulting in flaky tests which could be improved by using `StoppableWorkers` (which I hadn't anticipated, but the sign of a good design is that people want to use it in more ways than you planned). 

I'll try doing that here, and we'll see if it helps...